sysbuild/CMakeLists.txt

```diff
@@ -331,6 +336,10 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_post_cmake)
     include_mesh_dfu_metadata()
   endif()
 
+  if(SB_CONFIG_MATTER_FACTORY_DATA)
+    include_matter_factory_data()
```

Comment:
I can see that factory data hex is generated in the build directory but is not merged with the firmware. It seems that config `MATTER_FACTORY_DATA_MERGE_WITH_FIRMWARE` is not used anywhere. I found a problem, I've added a comment on a PR to sdk-connectedhomeip.

Reply:
Fixed